draft-ietf-scitt-architecture.md

```diff
@@ -320,28 +320,28 @@ Failure to produce this proof can indicate that the Transparency Services operat
                  .----+----.  .----------.  Identifiers
 Issuer      --> | Statement ||  Envelope  +<------------------.
                  '----+----'  '-----+----'                     |
-                      |             |           +--------------+---+
-                       '----. .----'            | Identity         |
-                             |                  | Documents        |
-                             v                  +-------+------+---+
-                        .----+----.                     |      |
-                       |  Signed   |    COSE Signing    |      |
-                       | Statement +<-------------------+      |
-                        '----+----'                     |      |
-                             |               +----------+---+  |
-                          .-' '------------->+ Transparency |  |
-                         |   .-------.       |              |  |
-Transparency -->         |  | Receipt +<-----+   Service    |  |
-     Service             |   '---+---'       +------------+-+  |
-                          '-. .-'                         |    |
-                             |                            |    |
-                             v                            |    |
-                       .-----+-----.                      |    |
-                      | Transparent |                     |    |
-                      |  Statement  |                     |    |
-                       '-----+-----'                      |    |
-                             |                            |    |
-                             |'-------.     .-------------)---'
+                      |             |            .-------------+--.
+                       '----. .----'            |    Identity      |
+                             |                  |    Documents     +-----.
+                             v                   '------+---------'       |
+                        .----+----.                     |                 |
+                       |  Signed   |    COSE Signing    |                 |
+                       | Statement +<-------------------+                 |
+                        '----+----'                     |                 |
+                             |               +----------+----+            |
+                          .-' '------------->+ Transparency  |            |
+                         |   .--------.      |               |            |
+Transparency -->         |  | Receipts +<----+   Services    |            |
+     Service             |   '---+----'      +------------+--+            |
+                          '-. .-'                         |               |
+                             |                            |               |
+                             v                            |               |
+                       .-----+-----.                      |               |
```

Comment:
```suggestion
                        '----+----'                     |                 |
                             |                  +-------+--------+        |
                          .-' '---------------->+                |        |
                         |   .--------.         |  Transparency  |        |
Transparency -->         |  | Receipt  +-.      |    Services    |        |
     Service             |   '+-------'  +<-----+                |        |
                         |    | Receipt  |      +---------+------+        |
                         |     '+-------'                 |               |
                          '-. .-'                         |               |
                             |                            |               |
                             v                            |               |
                       .-----+-----.                      |               |
```

*sigh* here my attempt at it then. Not that great, but it hopefully conveys the intent.

Reply:
this is again multiple receipts from the same TS... this is not very helpful.

The helpful thing, is to obtain multiple receipts from distinct TS (operated by different trusted authorities).

Reply:
while it says "services" on the right, I agree. Can that be compacted enough to be put into this diagram?

Reply:
I think the multiple receipts was a mistake. In our discussions, we wanted to represent multiple transparency services. Each issues a single receipt for each signed statement. 
```suggestion
                        '----+----'                     |                 |
                             |               +----------+----+            |
                          .-' '------------->+ Transparency  |            |
                         |   .--------.      |               |            |
Transparency -->         |  | Receipt  +<----+   Services    |            |
     Service             |   '---+----'      +------------+--+            |
                          '-. .-'                         |               |
                             |                            |               |
                             v                            |               |
                       .-----+-----.                      |               |
```

Reply:
@henkbirkholz, if you wanted to show the layered boxes around Transparency Services, instead of Receipt(s), that would work well

Reply:
@henkbirkholz : I wanted to show multiple Receipt instances and Multiple Transparency service instances, however the diagram became `clumsy`, so i sticked with plurality

Reply:
@henkbirkholz @OR13 @SteveLasker : I have corrected the diagram to be more explicit!

Please have a look now!

Reply:
@SteveLasker how are multiple receipts from different TS a mistake?

Reply:
Based on my understanding of our last discussion, we wanted to simplify #156 and agreed to show duplicity of Transparency Services. 
Yes, each transparency service generates a different receipt for the same signed statement placed upon each TS. 
Yes, one ore more entities can register the same signed statement on the same transparency service, producing multiple receipts for the same transparent statement. 
We have discussed a TS can re-generate the receipt, so thinking more, it could have a multiplicity of receipts. 
My gut is we should keep the first architecture overview relatively simple, letting readers digest the concepts. WIth incremental information, more details are exposed to the reader. 
Just my 2c.

Reply:
@henkbirkholz, the way @yogeshbdeshpande added multiple receipts, each from a different Transparency Service resolved my concerns.